pygfx/controls/_base.py

```diff
@@ -0,0 +1,50 @@
+from typing import Tuple
+
+from ..linalg import Vector3
+from ..utils.viewport import Viewport
+
+
+# todo: I hate that these classes are plural, maybe rename to Controller?
```

Comment:
I don't particularly mind if you rename :)

Reply:
I might do that in another PR :)